### ISSUE TITLE
Subsequent viewings of the depiction gives an error

### ISSUE THREAD

**Issue by bpluhar** [supermamon/Reposi3#13]
When viewing a package for the first time, all goes well. When navigating away from that view and back into it, I receive an error stating "Oh snap! The package you are trying to view is not hosted on this repository."

It's in the div with the id of packageError, but I'm not too sure what's making it appear. As a full-time backend developer, this really has me stumped.

I have verified that my info.xml's are all proper XML formatted, and even used: https://codebeautify.org/xmlvalidator to validate it as you stated in a previously listed issue.

**Comment by supermamon** [comment]
Does it happen both when viewing on both mobile and computer? Do you have a link to the repo where I can check?

**Comment by bpluhar** [comment]
Sorry for the late reply!

Big oops! I'm guessing there was a blip in the network since I was only getting that issue on LTE. Switched to WiFi when I got home from work and the issue was no longer happening. I had a buddy try it on his LTE network as well (same as me, T-Mobile), and he doesn't get that error that I mentioned.

I went ahead and rebooted my phone entirely, rejailbroke and I'm no longer getting that issue either. I'm not entirely sure what was causing it but it has seemed to iron itself out.

All is well!
